### PR TITLE
Re-adds some sound effects all the way back from 2015 + 2015 egun sprite

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: Kawaii-Big-Boss
soundadd: Some machine sounds have been reverted to their 2015 state.
imageadd: The Egun has its old sprite from 2015.
/:cl:

[why]: This is not a complete re-add of all of our older sprites, nor our older sounds, just the ones that immediately came to mind.
The following sounds have been changed:
Windoors
Announcements/Alerts
Airlocks
Spray Bottles
Punches
Lasers

Windoor Sound Change:
Because the old one just sounds more like a piece of shit. It's also 100% synced with the actual animation of the windoor.

Announcements/Alerts:
Simpler, quieter, but still to the point. Should no longer violate your ears with loud alarms

Airlocks:
Honestly I just prefer this one. Way back when, everyone loved the sound effect when it was first re-added.

Spray Bottles:
Much shorter sound effect, doesn't play the sound twice in a row, but still works.

Punches:
Older sound effects, simply put, had more of a punch to them. They're a bit quiet right now for reasons I don't want to tinker around with until Carbonhell gives me the okay.

Lasers:
See above.

### Egun Sprite Change

The older egun has a much better design to it. Unfortunately, the sprite has also aged quite a bit and doesn't have that same "polish" as the current egun.
old egun sprite
Personally I would still take it over the current one any day. It has better visual clarity when viewing in your hand slots (but the in-hands aren't changed), and the design is a lot less clean and refined. It, simply put, looks like a cheap or retro piece of crap that Nanotrasen would hand out for mall cop security.

In stark contrast, another sprite I wanted to add was a laser gun resprite, also from 2015 base (but was used for a different item)
old laser sprite
This sprite looks way more advanced and/or modern, but I can't quite tell if I like it over the current laser gun sprite, which is why I didn't add it in this PR. Also because Tortellini Tony's inevitable -1 and or ragging is already going to be annoying, don't need to give him more ammo :smug:

A video will go up soon demonstrating all of the new sounds.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -52,8 +52,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/list/inherent_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
 
 	var/attack_verb = "punch"	// punch-specific attack verb
-	var/sound/attack_sound = 'sound/weapons/punch1.ogg'
-	var/sound/miss_sound = 'sound/weapons/punchmiss.ogg'
+	// Hippie Start - Applies our punch sound effects from ye olde days.
+	var/sound/attack_sound = 'hippiestation/sound/weapons/punch1.ogg'
```
Comment (ghost) [inline code/modules/mob/living/carbon/human/species.dm:57]: can be modularised by making a new file

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -52,8 +52,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
...
+	var/sound/miss_sound = 'hippiestation/sound/weapons/punchmiss.ogg'
```
Comment (ghost) [inline code/modules/mob/living/carbon/human/species.dm:58]: can be modularised by making a new file

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by Carbonhell** [comment]
Sure +1

**Review by Carbonhell — CHANGES_REQUESTED** [review]
All the comments in tg files are one line or at most two line. Just do
code // hippie -- reason
No need for new lines, also be sure to use uncapitalized "hippie" and double -

**Comment by praisenarsie** [comment]
Old egun looks like somebody left it in an attic for 50+ years and when they found it, they accidently chipped off some of the material while dusting it off.

**Comment by DaydreamPuppeh** [comment]
The old egun Sprite is better than the current one. +1, though I want the Sprite to be updated to fit the other sprites

**Comment by Eagle-Eyess** [comment]
+1

**Comment by earthmoonjune** [comment]
egun sprite is eh, laser gun sprite is good. +1 tho

**Comment by yskefunky** [comment]
i'd like if you'd be able to make it a small percentage just to change things up a bit for the newer sounds, like how we have doom sounds rarely play. also make the new sprite egun another weapon for mappers to use as a nice loot drop but otherwise +1

**Comment by yobob591** [comment]
+1 PEW PEW

**Comment by ghost** [comment]
cool af

**Comment by danhibiki** [comment]
+2 headmins vote this should be good

**Comment by GuyonBroadway** [comment]
This still requires modularization.

**Comment by Manly-Dwarf** [comment]
i don't wanna modularize this it's BORING

**Comment by ghost** [comment]
it should be easy to modularize, also make the new egun a premium egun from the security vendor for 200 bucks

**Comment by bardofrage** [comment]
+1 again

**Comment by earthmoonjune** [comment]
fking modularize you lazy bastard.................

**Comment by k-k-karma** [comment]
Reopen when this is modularized.

**Pull request closed without merging**